daemon/src/maker_cfd.rs

```diff
@@ -68,21 +71,21 @@ impl Actor {
         db: sqlx::SqlitePool,
         wallet: Wallet,
         oracle_pk: schnorrsig::PublicKey,
-        cfd_feed_actor_inbox: watch::Sender<Vec<Cfd>>,
+        cfd_feed: CfdFeed,
         order_feed_sender: watch::Sender<Option<Order>>,
         takers: Address<maker_inc_connections::Actor>,
         monitor_actor: Address<monitor::Actor<Actor>>,
     ) -> Result<Self> {
         let mut conn = db.acquire().await?;
 
         // populate the CFD feed with existing CFDs
-        cfd_feed_actor_inbox.send(load_all_cfds(&mut conn).await?)?;
+        cfd_feed.update(&mut conn).await?;
```

Comment:
I think this should happen outside of the `Actor` actually, there is no reason why we can't initialize the feed already with the list of CFDs.

Reply:
I'll leave this optimisation out for now, as it's the same behaviour as before. can come back to it later.